dapp-developers/developer-toolkit/ethereum-api/dev-env/.pages

```diff
@@ -2,6 +2,6 @@ title: Dev Environments
 hide: false
 nav:
   - index.md
-  - remix.md
-  - hardhat.md
-  - foundry.md
+  - 'Deploy Contracts with Remix': remix.md
+  - 'Deploy Contracts with Hardhat': hardhat.md
+  - 'Deploy Contracts with Foundry': foundry.md
```

Comment:
thank you for doing this!! i guess i didn't even realize that these were all using "Deploy Contracts with X" in the left nav- can you please change them to be "Remix", "Hardhat", and "Foundry"? so it aligns with the rest of the sections, where we just list the tool name